src/backend/main.ts

```diff
@@ -256,7 +256,11 @@ const loadMainWindowURL = function () {
       `file://${path.join(publicDir, '../build/index.html?view=App')}`
     )
     if (!isMac) {
```

Comment:
We should remove this check. On heroic it was giving an error message since we do not have a certificate for signing, but here should be fine